lwr_controllers/Stiffness k_FRI
geometry_msgs/Pose x_FRI
geometry_msgs/Wrench f_FRI
std_msgs/Float64[6] d_FRI